llarp/constants/path.hpp

```diff
@@ -31,9 +31,9 @@ namespace llarp
     constexpr auto build_timeout = 30s;
 
     /// measure latency every this interval ms
-    constexpr auto latency_interval = 5s;
+    constexpr auto latency_interval = 20s;
```

Comment:
This is a big change, would you mind briefly describing the rationale?

Reply:
reduce idle bandwidth useage